Package.resolved

```diff
@@ -5,17 +5,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/getditto/DittoSwiftPackage",
       "state" : {
-        "revision" : "ddb45fb769c0dea365d192dfb5d5641aae396035",
-        "version" : "3.0.7"
+        "revision" : "63db2b032b5d3f6505ead0546e866a2caab401e2",
+        "version" : "4.0.0"
```

Comment:
Can we update the resolved version to 4.4.4?

Reply:
No, we want to keep resolved at 4.0.0 to keep dependencies as broadly defined as possible. For the package this is important. Devs can update package versions locally and have more narrowly defined dependencies local to their apps.